api-service/service/hosts.go

```diff
@@ -68,16 +74,37 @@ func (as *APIService) SearchHostsAsLMS(filters dto.SearchHostsAsLMS) (*excelize.
 			}
 			if j == 4 {
 				indexsheetHostAdded := lms.NewSheet(sheetHostAdded)
-				iindexSheetDatabaseEbsDbTier := lms.GetSheetIndex(sheetDatabaseEbsDbTier)
-				errs := lms.CopySheet(iindexSheetDatabaseEbsDbTier, indexsheetHostAdded)
+				indexSheetDatabaseEbsDbTier := lms.GetSheetIndex(sheetDatabaseEbsDbTier)
+				errs := lms.CopySheet(indexSheetDatabaseEbsDbTier, indexsheetHostAdded)
 				if errs != nil {
 					return nil, errs
 				}
-				lms.SetActiveSheet(iindexSheetDatabaseEbsDbTier)
+				lms.SetActiveSheet(indexSheetDatabaseEbsDbTier)
 			}
 			setCellValueLMS(lms, sheetHostAdded, j, csiByHostname, val)
 			j++
 		}
+
+		//HostDismissed management
+		if (filters.From != utils.MIN_TIME ||
+			filters.To != utils.MAX_TIME) &&
+			dismissedAt.After(filters.From) &&
+			dismissedAt.Before(filters.To) {
+			if z == 0 {
+				z = 4 // offset for headers
+			}
+			if z == 4 {
```

Comment:
If z was 0, you already know it'll be 4 here, am I right?
So can  you remove this check and do what you are doing here in the above `if`?